website/docs/r/service_account.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_service_account"
+sidebar_current: "docs-vcd-resource-service_account"
+description: |-
+  Provides a resource to manage Service Accounts. Service Accounts can have defined roles
+  and act just like a VCD user. Service Accounts, when activated, provide one-time use
+  access tokens for authentication to the VCD API, during which a new access token is generated.
+---
+
+# vcd\_service\_account 
+
+Supported in provider *v3.10+* and VCD 10.4+.
+
+Provides a resource to manage Service Accounts. Service Accounts can have defined roles
+and act just like a VCD user. Service Accounts, when activated, provide one-time use
+access tokens for authentication to the VCD API, during which a new access token is generated.
+Explained in more detail [here].[service-accounts]
+
+## Example Usage 
+
+```hcl
+data "vcd_role" "vapp_author" {
+  org  = "my-org"
+  name = "vApp Author"
+}
+
+resource "vcd_service_account" "example_service" {
+  org  = "my-org"
+  name = "example"
+  role = data.vcd_role.vapp_author.id
+
+  software_id      = "12345678-1234-1234-1234-1234567890ab"
+  software_version = "1.0.0"
+  uri              = "example.com"
+
+  file_name = "example_service.json"
+
+  active = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful
+  when connected as sysadmin working across different organisations.
+* `name` - (Required) A unique name for the Service Account
+* `role` - (Required) ID of a Role.
+* `software_id` - (Required) UUID of the Service Account.
```

Comment:
Shall we specify that this ID depends on the user and can be any UUID?

Reply:
Updated.